src/app/pages/support/support.html

```diff
@@ -9,13 +9,7 @@
 
 <ion-content>
   <div class="support-logo">
-    <img
-      ngSrc="assets/img/appicon.svg"
-      alt="Ionic Logo"
-      width="200"
-      height="200"
-      priority
-    />
```

Comment:
This image was conflicting with its CSS and causing a ratio conflict. In this change, I tried to make it act like the support logos in its neighboring pages.